src/components/contextual/pages/pool/MyPoolBalancesCard.vue

```diff
@@ -119,7 +119,7 @@ const showMigrateButton = computed(
  * METHODS
  */
 function weightLabelFor(address: string): string {
-  return fNum2(props.pool.onchain.tokens[address].weight, {
+  return fNum2(props.pool?.onchain?.tokens?.[address]?.weight || '0', {
```

Comment:
Is this just null while it's loading? If so I think it should return '-' instead of '0' like `weightFor` in DefaultPool.vue does. Or at least these two functions should return the same value when it doesn't exist.